launch/demo_rviz.launch.py

```diff
@@ -1,3 +1,32 @@
+# Copyright 2021 PickNik Inc.
```

Comment:
It's an old discussion, with a relatively new touch in ROS2. Do we *really* want copyright texts in all **launch** files?
This clutters otherwise mostly trivial files (granted, the ROS2 files do not look that trivial at the moment...)

I would vote to leave them out there unless it's impossible to silence some automatic pedantic checker.

Reply:
Unfortunately, I'm not aware of a way to exclude certain files from ament_copyright linter, removing the copyright from this file will cause the linter to report failure

Reply:
But how would we use it with `ament_lint_auto_find_test_dependencies` .? we either need to remove it and explicitly list all linters' cmake test ourselves (I tried this and it will complicate the `BUILD_TESTING` section) or disable it and create a pre-commit hook to handle having exclude argument

Reply:
> But how would we use it with ament_lint_auto_find_test_dependencies .?

If the macro does not provide a way for that already, it might be worthwhile to add a cmake variable to the command (`AMENT_COPYRIGHT_OPTIONS`) upstream to set the behavior in another line before the call?

Reply:
> it might be worthwhile to add a cmake variable to the command (AMENT_COPYRIGHT_OPTIONS)

I like this idea, added to my todo list